Description: Disable unavailalbe optional feature and dep, jiff-02 (jiff 0.2)
 which Debian has only 0.1, and the explicit 02 mark indicates 0.1 is
 insufficient
Forwarde: not-needed
Author: Blair Noctis <ncts@debian.org>
Last-Update: 2025-03-24
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -100 +100 @@
-jiff-02 = ["dep:jiff"]
+#jiff-02 = ["dep:jiff"]
@@ -169 +169 @@
-[dependencies.jiff]
+[disabled.dependencies.jiff]
